pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/PersistentTopics.java

```diff
@@ -775,15 +775,20 @@ public void skipAllMessages(@PathParam("property") String property, @PathParam("
             }
         } else {
             validateAdminOperationOnDestination(dn, authoritative);
-            PersistentTopic topic = getTopicReference(dn);
+            if (!(getTopicReference(dn) instanceof PersistentTopic)) {
```

Comment:
Why do we need to check if it's persistent topic? Shouldn't this already be guaranteed by having this handler on `/admin/persistent/...` ?